lib/jekyll/compose.rb

```diff
@@ -2,6 +2,8 @@
 
 module Jekyll
   module Compose
+    TYPE = "md"
+    LAYOUT = "post"
```

Comment:
Let's call this `DEFAULT_TYPE` and `DEFAULT_LAYOUT`. And make sure `jekyll-compose.rb` requires this file (or stick them there)